### PR TITLE
ci: Run a weekly cargo-audit job

### PR DESCRIPTION
**Pull request by Swatinem**

### CODE REVIEW

File: .github/workflows/weekly.yml [hunk 1]
```diff
@@ -27,8 +29,7 @@ jobs:
           override: true
           components: clippy
 
-      - name: Run clippy ${{ matrix.rust }}
```
Comment (relaxolotl) [inline .github/workflows/weekly.yml:30]: any reason why we can't keep this name?

File: .github/workflows/weekly.yml [hunk 2]
```diff
@@ -37,3 +38,14 @@ jobs:
         with:
           command: test
           args: --workspace --all-features
+
+  weekly-audit:
+    name: Audit
```
Comment (relaxolotl) [inline .github/workflows/weekly.yml:43]: any reason why this can't be "security audit"?

**Review by relaxolotl — APPROVED** [review]
Something to keep in mind is that `actions-rs/audit-check` hasn't seen much activity from its author for over a year (last release was May 2020). If the default settings work well enough for our needs then this shouldn't be a cause for concern.

**Comment by Swatinem** [comment]
I just kept the job and naming the same as with the sentry-rust repo.

**Pull request merged**